### DISCUSSION TITLE
Demo Not Working?

### DISCUSSION THREAD

**Discussion by TronCarter269** [rzeldent/esp32-smartdisplay#206] [General]
I could very well be doing something wrong, but today I cloned the demo git repo to a local folder, opened the folder in PlatformIO, uncommented the board I'm using in platformio.ini, saved it and tried to send it to my board.  Blank screen and no LED activity.

The only thing I can see is that the demo was written for LVGL 9.1 and PlatformIO seems to be installing 9.2.  Any ideas?

`Library Manager: esp32_smartdisplay@3.0.0+sha.08d0ce2 has been installed!

Library Manager: Resolving dependencies...

Library Manager: Installing lvgl/lvgl @ ^9.2.0

Unpacking  [####################################]  100%

Library Manager: lvgl@9.2.0 has been installed!
`

**Comment by rzeldent** [comment]
I suspect you try to run the esp32-smartdisplay, not the esp32-smartdisplay-demo repository.
Esp32-smartdisplay is just a library that can be included in a project.

**Comment by TronCarter269** [comment]
No, I'm using the demo repo.
![2024-10-22 15_55_27-platformio ini - esp32-smartdisplay-demo - Visual Studio Code](https://github.com/user-attachments/assets/cd271629-882d-4001-8d39-33c916f55128)

**Comment by TronCarter269** [comment]
As you can see from the above screenshot, I tried to force v9.1, but that didn't work.  Compile errors.

**Reply by rzeldent** [reply]
LVGL9.1 is a dependency of the library, Adding this does not work,

**Reply by TronCarter269** [reply]
OK, but I’m still not sure what I’m doing wrong.

**Comment by TronCarter269** [comment]
FOUND IT!!  Well, actually another user found the problem and I stumbled upon it in another discussion thread:

[https://github.com/rzeldent/esp32-smartdisplay-demo/issues/43#issuecomment-2336613018](url)

` "'-D LVGL_BUFFER_PIXELS=(DISPLAY_WIDTH*DISPLAY_HEIGHT/4)'",`

Needs to be changed in boards\esp32-3248S035C.json to:

` "'-D LVGL_BUFFER_PIXELS=(DISPLAY_WIDTH*DISPLAY_HEIGHT/16)'",`

This is for board ESP32-3248S035C, so I propose that the board definition be changed.
